oleesir/Banking-app
# #164727170Admin should be able to see delete a specific account page.

#### What does this PR do?
implement delete-account page
#### Description of Task to be completed?
implement delete-account page
#### How should this be manually tested?
* Navigate to the UI folder and click on delete-account.html in your browser
* You will see delete-accounts-template
#### What are the relevant pivotal tracker stories?
[#164727170](https://www.pivotaltracker.com/story/show/164727170)
#### Any background context you want to add?
N/A
#### Screenshots
![Uploading Screenshot 2019-03-26 at 10.33.49 PM.png…]()
